config/config-example.js

```diff
@@ -408,6 +408,14 @@ exports.grouplist = [
 		broadcast: true,
 	},
 	{
+		symbol: '♿',
+		id: "retired",
+		name: "Retired",
+		inherit: ' ',
```

Comment:
The idea is to have it below the Voice rank seeing as it's essentially reg but with broadcast access